uniffi/src/interface/mod.rs

```diff
@@ -699,6 +845,10 @@ impl APIConverter<Method> for weedle::interface::OperationInterfaceMember<'_> {
                 weedle::types::ReturnType::Type(t) => Some(t.resolve_type_definition(ci)?),
             },
             ffi_func: Default::default(),
+            attributes: match &self.attributes {
+                Some(attr) => Attributes::try_from(attr)?,
```

Comment:
This is missing erroring for valid Attributes that are not valid for methods (it errors out if attributes are completely invalid, but might want to ensure that the attributes are valid for methods, like `Error` for example should only be on `Enum`s